transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/NumberWidget.java

```diff
@@ -83,6 +83,7 @@ public void onFocusChange(View v, boolean hasFocus) {
             }
 
             editText.setOnKeyListener(new DefaultKeyListener(mDefaultFocusView, editText));
+            editText.addTextChangedListener(new TextChangeListener());
```

Comment:
NumberWidget already has a `TextChangedListener` why assign a new one?

Reply:
https://developer.android.com/reference/android/view/View.OnKeyListener
OnKeyListener not triggered in some keyboard, google recommended to use textchange listener for text changes.

Reply:
fixed